### ISSUE TITLE
Contribution counter does not increase

### ISSUE THREAD

**Issue by Tealk** [cheeaun/phanpy#571]
**Describe the bug**
The thread counter is not incremented.
- Which site: https://phanpy.rollenspiel.social
- Which site version: 2024.06.03.0a6030c
- Which instance: https://rollenspiel.social

**Expected behavior**
The counter should start from 1 and count up as many posts as there are.

**Screenshots**
https://rollenspiel.social/@Igni/112607984951868590
![image](https://github.com/cheeaun/phanpy/assets/12276250/ac1c294b-5ec7-42c1-9a29-cb99c10d5c2b)

**Desktop (please complete the following information):**
 - OS: Fedora 40
 - Browser Firefox
 - Version 126.0.1 (64-bit)

**Comment by cheeaun** [comment]
Hmmm… the rest of the posts are replies to the 1st post, instead of chained replies.

**Comment by Fastidious** [comment]
> Hmmm… the rest of the posts are replies to the 1st post, instead of chained replies.

How are threads formed? I thought it was by creating a post, and then replying to it with subsequent posts, over and over, until the last on the "thread". Isn't that the case?

**Comment by cheeaun** [comment]
@Fastidious yes, it's a reply of a reply of a reply (and goes on). Above use-case if what I call "forked" threads; if there are two replies by same author under the author's 1st post, there's no order on which one is 2nd or 3rd.

Possible solution would be sort by posted time, but again, we wouldn't be able to guess the real intentions here — latest might not be the 3rd and it gets worse if there are multiple replies under multiple replies under the post (and goes on) 😩

**Comment by Fastidious** [comment]
@Tealk in case you haven’t gotten a notification, this is an user error. See reply above.

**Comment by Tealk** [comment]
yes, i have read it, but i think i can't educate the users for it.
